db/model.js

```diff
@@ -1 +1,5 @@
-module.exports.endpoints = require("../endpoints.json")
\ No newline at end of file
+const db = require("./connection")
+
+module.exports.endpoints = require("../endpoints.json")
+
+module.exports.topics = require("./data/development-data/topics.js")
```

Comment:
We want to make sure that we are sending the right data when we request this endpoint - at the moment you are always sending dev data regardless of when/where/how the request is made - but we should be sending test data when we run the tests. This is why we should instead be making a query to get this data from the database as we are programatically connecting to the right database containing the right type of data every time a request is made, rather than sending the information that is stored locally. We only did this with endpoints.json as this information was not stored on the database